geocity/apps/accounts/templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base_generic.html" %}
+{% load bootstrap4 %}
+{% load static %}
+{% load i18n %}
+
+{% block content %}
+
+{% if validlink %}
+
+<h2>Geocity</h2>
+<hr>
+<h3>{% translate "Choisissez un nouveau mot de passe!" %}</h3>
```

Comment:
```suggestion
<h3>{% translate "Choisissez un nouveau mot de passe" %}</h3>
```

Reply:
And on the default site?

Reply:
The default site should have a name...but another issue related to default site is opened

Reply:
https://github.com/yverdon/geocity/issues/525